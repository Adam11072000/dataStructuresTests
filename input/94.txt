init 90
addImage 248
addImage 129
deleteImage 248
quit