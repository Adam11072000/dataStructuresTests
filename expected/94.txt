init done.
addImage: SUCCESS
addImage: SUCCESS
deleteImage: SUCCESS
quit done.
